### DISCUSSION TITLE
Input / select without label

### DISCUSSION THREAD

**Discussion by marektonik** [twilio-labs/paste#3343] [Q&A] [answered]
Hey, is there an option to use input or select without the label? There are some cases, where label is not needed, although I can't see variant without it.

Examples where input/select without label is needed:

<img width="706" alt="Zrzut ekranu 2023-07-24 o 15 20 53" src="https://github.com/twilio-labs/paste/assets/140410476/b4a9cce6-94d5-4bf9-87f5-68234f3eb596">
<img width="461" alt="Zrzut ekranu 2023-07-24 o 15 20 16" src="https://github.com/twilio-labs/paste/assets/140410476/e83d1ba6-25e2-40ef-8df3-dbdf7aff20d4">
<img width="670" alt="Zrzut ekranu 2023-07-24 o 15 19 59" src="https://github.com/twilio-labs/paste/assets/140410476/9e62a109-399d-41eb-87c2-2030b0d0973c">

**Comment by serifluous** [comment, marked as answer]
Hey @marektonik ! All form fields should have a label according to WCAG: https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=332#labels-or-instructions, with the exception of search fields because they're a very common interaction pattern across the web.

For those last 2 situations, those form fields will need a label added to them because it's difficult to understand visually what they're controlling.

**Reply by marektonik** [reply]
@serifluous understand, thank you!
